docs/UserGuide.md

```diff
@@ -100,180 +93,245 @@ Do have a look at our Table of Contents to locate the help you need!
 
 </div>
 
---------------------------------------------------------------------------------------------------------------------
+As you explore the features that SociaLite has to offer, look out for these boxes:
 
-## Basic Functionality <a name="basic_functionality"></a>
+This box tells you the acceptable fields that you can provide for a particular command:
 
-### Adding a person: `add`
+<div markdown="block" class="alert alert-primary">
+**:mag_right: Format:**
+`add n/NAME p/PHONE_NUMBER [t/TAG]…​ [date/NAME:YYYY-MM-DD[:monthly|:yearly]]…​ [fb/FACEBOOK] [ig/INSTAGRAM] [tele/TELEGRAM] [tiktok/TIKTOK] [twitter/TWITTER]`
+* Detail 1
+* Detail 2
+* Detail 3
+</div>
 
-Adds a person to the address book.
+If you still aren't sure of how a particular command works, this box shows you examples that you can try out:
 
-Format: `add n/NAME p/PHONE_NUMBER [t/TAG]…​ [date/NAME:YYYY-MM-DD[:monthly|:yearly]]…​ [fb/FACEBOOK] [ig/INSTAGRAM] [tele/TELEGRAM] [tiktok/TIKTOK] [twitter/TWITTER]`
+<div markdown="block" class="alert alert-success">
+**:heavy_check_mark: Example:** `add n/Alex Yeoh p/87438807 t/friends fb/alex.yeoh ig/alex.yeoh tele/alyeoh tiktok/alex.yeoh date/birthday:2000-01-01:yearly`
 
-<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-A person can have any number of tags and dates (including 0).
+Adds a new contact called "Alex Yeoh" with phone number "87438807" to SociaLite.
 </div>
 
-Examples:
-* `add n/Alex Yeoh p/87438807 t/friends fb/alex.yeoh ig/alex.yeoh tele/alyeoh tiktok/alex.yeoh date/birthday:2000-01-01:yearly`
-* `add n/Charlotte Oliveiro p/93210284 t/neighbours fb/charlotte.oli tele/olichar twitter/charolive date/wedding anniversary:2020-01-01:yearly`
+--------------------------------------------------------------------------------------------------------------------
 
-<br>
+## Commands <a name="commands"></a>
+
+This section provides you with comprehensive details of all the commands available on SociaLite. 
+Simply click on any command below to learn more about it!
+
+| `COMMAND` | Description |
+| --------- | ----------- |
+| [`add`](#add) | Add a contact |
+| [`list`](#list) | List all contacts |
+| [`CLICK`](#click) | Access a contact's social media page |
+| [`↑`/`↓`](#scroll) | Track history of commands |
+| [`edit`](#edit) | Edit a contact |
```

Comment:
For this command, since there are many options that also include the next 3 lines, should we name it to something like `edit ...`? (might not be so appropriate here haha)

Reply:
Sure! Should I make the same change to `add` or just leave it as it is?

Reply:
Yep, missed out on add, but maybe for add we can do something like `add n/NAME p/PHONENUMBER ...` or the like! Not exactly sure on the format hahah

Reply:
Suggestions adopted in updated Commands Table